src/locale/locales/it/messages.po

```diff
@@ -3581,7 +3581,7 @@ msgstr "Piace A"
 msgid "Likes"
 msgstr "Mi piace"
 
-#: src/view/com/post-thread/PostThreadItem.tsx:217
+#: src/view/com/post-thread/PostThreadItem.tsx:212
 msgid "Likes on this post"
 msgstr "Mi Piace in questo post"
```

Comment:
"Mi piace in questo post"

Reply:
@franciscomrtt what about joining the Transifex project? That way you don't have to suggest me changes: you can do everything youself 😉
If so, you can simply provide me your Transifex username

Reply:
Sarebbe fantastico poter contribuire al progetto. Sono nuovo su GitHub e non so benissimo come usarlo, ti ringrazio per il suggerimento. Sarò a breve su Transifex con lo stesso nickname!

Reply:
Ottimo! Fammi sapere quando hai creato l'account su Transifex

Reply:
Ho creato il profilo e penso di essere riuscito a mandare direttamente richiesta per la partecipazione al progetto.

Reply:
Richiesta approvata 👍

Reply:
Ho dato un’occhiata alla maggior parte delle stringhe e ho apportato tante piccole modifiche: maiuscole e spazi errati, traduzione di frasi con “più contesto”; dagli un’occhiata, ho cercato di notare anche il più piccolo dei dettagli.